main/src/ca/uwaterloo/flix/tools/pkg/FlixPackageManager.scala

```diff
@@ -15,59 +15,89 @@
  */
 package ca.uwaterloo.flix.tools.pkg
 
-import ca.uwaterloo.flix.tools.Packager
 import ca.uwaterloo.flix.tools.Packager.getLibraryDirectory
+import ca.uwaterloo.flix.tools.pkg.Dependency.FlixDependency
 import ca.uwaterloo.flix.tools.pkg.github.GitHub
-import ca.uwaterloo.flix.util.Result.ToOk
-import ca.uwaterloo.flix.util.{Options, Result}
+import ca.uwaterloo.flix.util.Result.{Err, Ok, ToOk}
+import ca.uwaterloo.flix.util.Result
 
-import java.io.File
+import java.io.{IOException, PrintStream}
 import java.nio.file.{Files, Path, StandardCopyOption}
 import scala.util.Using
 
 object FlixPackageManager {
 
   // TODO: Move functionality from "Packager" in here.
 
+  //TODO: tests - how?
   /**
-    * Installs a flix package from the Github `project`.
+    * Installs all the Flix dependencies for a Manifest.
+    */
+  def installAll(manifest: Manifest, path: Path)(implicit out: PrintStream): Result[Unit, PackageError] = {
```

Comment:
Perhaps we could return the packages/dependencies that were freshly downloaded in a list?